geode-core/src/distributedTest/java/org/apache/geode/management/ClientHealthStatsDUnitTest.java

```diff
@@ -12,352 +12,208 @@
  */
+
 package org.apache.geode.management;
 
-import static java.util.concurrent.TimeUnit.MINUTES;
 import static org.apache.geode.distributed.ConfigurationProperties.DURABLE_CLIENT_ID;
 import static org.apache.geode.distributed.ConfigurationProperties.DURABLE_CLIENT_TIMEOUT;
-import static org.apache.geode.distributed.ConfigurationProperties.STATISTIC_SAMPLING_ENABLED;
-import static org.apache.geode.test.dunit.Host.getHost;
-import static org.apache.geode.test.dunit.IgnoredException.addIgnoredException;
-import static org.apache.geode.test.dunit.Invoke.invokeInEveryVM;
-import static org.apache.geode.test.dunit.NetworkUtils.getServerHostName;
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.io.IOException;
 import java.io.Serializable;
-import java.util.Collection;
-import java.util.Properties;
-
-import javax.management.ObjectName;
 
 import org.awaitility.Awaitility;
-import org.awaitility.core.ConditionFactory;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 
-import org.apache.geode.cache.Cache;
 import org.apache.geode.cache.EntryEvent;
 import org.apache.geode.cache.Region;
-import org.apache.geode.cache.RegionFactory;
 import org.apache.geode.cache.RegionShortcut;
-import org.apache.geode.cache.client.ClientCache;
-import org.apache.geode.cache.client.ClientCacheFactory;
 import org.apache.geode.cache.client.ClientRegionFactory;
 import org.apache.geode.cache.client.ClientRegionShortcut;
-import org.apache.geode.cache.server.CacheServer;
 import org.apache.geode.cache.util.CacheListenerAdapter;
-import org.apache.geode.distributed.DistributedMember;
 import org.apache.geode.internal.cache.tier.sockets.CacheClientNotifier;
 import org.apache.geode.internal.cache.tier.sockets.CacheClientProxy;
-import org.apache.geode.management.internal.SystemManagementService;
-import org.apache.geode.test.dunit.VM;
-
-/**
- * Distributed tests for client stats exposed via {@link CacheServerMXBean}:
- * <ul>
- * <li>{@link CacheServerMXBean#showClientStats}
- * <li>{@link CacheServerMXBean#showAllClientStats}
- * <li>{@link CacheServerMXBean#showClientQueueDetails}
- * </ul>
- */
+import org.apache.geode.test.dunit.rules.ClientVM;
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.rules.VMProvider;
 
-@SuppressWarnings({"serial", "unused"})
 public class ClientHealthStatsDUnitTest implements Serializable {
-
   private static final int NUMBER_PUTS = 100;
 
-  private static final String KEY1 = "KEY1";
-  private static final String KEY2 = "KEY2";
-  private static final String VALUE1 = "VALUE1";
-  private static final String VALUE2 = "VALUE2";
+  @Rule
+  public ClusterStartupRule cluster = new ClusterStartupRule();
 
-  private static final String REGION_NAME =
-      ClientHealthStatsDUnitTest.class.getSimpleName() + "_Region";
+  private MemberVM locator, server;
```

Comment:
Declaring multiple variables on one line has never been part of our style. I think you should separate them.